docs/src/lib/functions.md

```diff
@@ -8,87 +8,116 @@ CurrentModule = DataFrames
 Pages = ["functions.md"]
 ```
 
-## Joining, Grouping, and Split-Apply-Combine
+## Constructing data frames
+```@docs
+copy
+DataFrame!
+similar
+```
 
+## Summary information
 ```@docs
-innerjoin
-leftjoin
-rightjoin
-outerjoin
-antijoin
-semijoin
-crossjoin
-combine
-groupby
-groupindices
-groupcols
-valuecols
-keys
-get
-stack
-unstack
+describe
+length
+ncol
+ndims
+nrow
+show
+size
 ```
 
-## Basics
+## Working with column names
+```@docs
+names
+propertynames
+rename
+rename!
+```
 
+## Mutating and transforming data frames and grouped data frames
 ```@docs
-allowmissing
-allowmissing!
 append!
-categorical
-categorical!
-completecases
-copy
-DataFrame!
-delete!
-describe
-disallowmissing
-disallowmissing!
-dropmissing
-dropmissing!
-eachcol
-eachrow
-filter
-filter!
+combine
 flatten
 hcat
 insertcols!
-length
 mapcols
 mapcols!
-names
-ncol
-ndims
-nonunique
-nrow
-order
 push!
-rename
-rename!
 repeat
 repeat!
 select
 select!
-show
-size
-sort
-sort!
 transform
 transform!
-unique!
 vcat
 ```
 
-## Unsorted
+## Reshaping data frames between tall and wide formats
+```@docs
+stack
+unstack
+```
+
+## Sorting
+```@docs
+issorted
+order
+sort
+sort!
+sortperm
+```
+
+## Joining
+```@docs
+antijoin
+crossjoin
+innerjoin
+leftjoin
+outerjoin
+rightjoin
+semijoin
+```
+
+## Grouping
+```@docs
+get
+groupby
+groupcols
+groupindices
+keys
+parent
+valuecols
+```
 
+## Filtering rows
 ```@docs
+completecases
+delete!
+dropmissing
+dropmissing!
+filter
+filter!
 first
 last
+nonunique
 unique
-propertynames
-similar
-sortperm
+unique!
+```
+
+## Changing column types
+```@docs
+allowmissing
+allowmissing!
+categorical
+categorical!
+disallowmissing
+disallowmissing!
+```
+
+## Iterating data frames
```

Comment:
`pairs` also works on GDF, right?

Reply:
right - and also e.g. for `DataFrameRow`, but I have put it here as we only provide a docstring for `pairs` only for `DataFrameColumns` (where it is not obvious what it produces).

Reply:
Maybe mention GDF in the heading then?

Reply:
ok